<launch>
    <!-- 加载参数文件 -->
    <rosparam command="load" file="$(find astar_node)/config/astar.yaml" />

    <!-- 启动规划节点 -->
    <node pkg="astar_node" type="astar_node" name="astar_node" output="screen">
    </node>
    
    <!-- (可选) 启动 Rviz 可视化 -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find astar_node)/rviz/planner.rviz" />
</launch>